### DISCUSSION TITLE
PNP modern search v3.8.0, unable to extract selected search refiners

### DISCUSSION THREAD

**Discussion by krishnare56** [microsoft-search/pnp-modern-search#3855] [Q&A]
Hi Team,

I am using pnp modern search v3.8.0

I have a SharePoint page where i have inserted search box, search results, search refiners. based on the selected refiners the search results are getting displayed,  
But i have a to show the selected search refiners in search results (not under search refiners), 
Need to display it in search results webpart but i am unable to find the property which have selected search refiners. can you guide me how can i capture selected search refiners in search results webpart.

i can see search refiners are added to query parameter in v4, but i cant find any solution in v3.8.0. guide me how i can achieve in v3.8.0

**Comment by wobba** [comment]
We don't do updates on v3. The last release was v3.23.0 back in 2021 I believe. https://github.com/microsoft-search/pnp-modern-search/releases/tag/v3.23.0.

I advice to uninstall v3 and reconfigure everything with v4.

**Reply by krishnare56** [reply]
Hey Wobba sorry, We cant upgrade to v4 present can you suggest me how we can get selected filters in v3.8.0.

**Reply by kasperbolarsen** [reply]
Version 3.x has been discontinued for years so you really should start upgrading to the most recent version 4.x
We can only support the most recent version due to time constraints
